packages/orchestrator/lib/clients/client.ts

```diff
@@ -270,7 +270,8 @@ export class OrchestratorClient {
                 type: 'on-event' as const
             }
         };
-        const res = await this.immediate(schedulingProps);
+
+        const res: Result<any, ClientError> = props.async ? await this.immediate(schedulingProps) : await this.immediateAndWait(schedulingProps);
```

Comment:
we actually don't care about the result value. Using `any` to please the typescript god